### DISCUSSION TITLE
v.0.10.0-beta

### DISCUSSION THREAD

**Discussion by awtkns** [reworkd/AgentGPT#1091] [Announcements]
![banner](https://github.com/reworkd/AgentGPT/assets/32209255/49c21a10-47c9-4d19-86d1-253f0527affb)
<h1 align="center">
🎉🤖&nbsp;&nbsp;&nbsp;Beta 0.10.0&nbsp;&nbsp;&nbsp;🤖🎉
</h1>

Today, we're thrilled to share an exciting release of AgentGPT 0.10.0. We are proud to finally announce that AgentGPT is supported by both YCombinator and Panache Venture.

However, even as we celebrate this milestone, we want to reaffirm our unwavering dedication to the open-source community. This commitment isn't just a component of our project; it's the very foundation on which we stand.

With the backing of YCombinator and Panache Venture, we're now in a position to truly push the boundaries of open-source AI agents. Their support will enable us to elevate AgentGPT to new heights, to further develop and improve on what we've already achieved.

Interested in the details of our journey with YCombinator? You can find out more [here](https://www.ycombinator.com/launches/J1r-reworkd-ai-the-open-source-zapier-of-ai-agents).

Join us in building the future of open source AI Agent automation ❤️ 

## ✨ Highlights
- 🎨 Brand new [landing page](https://reworkd.ai/). Thanks for making this possible @Jshen123 @jasangill1 @shahrishabh7 
- 🐛 Fixes to mypy, python typing, and setup scripts
- 📨 A new [blog](https://reworkd.ai/blog), so you can stay updated on all things GenAI
- 🤌 Multiple styling updates, usability, and accessibility improvements
- 🔥 Implementation of our brand new workflow execution engine

## ❤️ New Contributors
* @Gsciarra made their first contribution in https://github.com/reworkd/AgentGPT/pull/987
* @shahrishabh7 made their first contribution in https://github.com/reworkd/AgentGPT/pull/1085

## 👀 What's Changed
* ♻️Refactor highlight logic by @awtkns in https://github.com/reworkd/AgentGPT/pull/983
* Fix/failed to read dockerfile/issue 984 by @Gsciarra in https://github.com/reworkd/AgentGPT/pull/987
* 📚 Add Discord to footer by @awtkns in https://github.com/reworkd/AgentGPT/pull/985
* fix ergomake by @vieiralucas in https://github.com/reworkd/AgentGPT/pull/989
* 😍 Update Logos by @awtkns in https://github.com/reworkd/AgentGPT/pull/986
* ✨ Docs Formatting by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/993
* ✨ Fixed Loss of Agent Data upon Login by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/991
* 🦴 Skeleton landing page by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/1000
* 💬 1003 update hero section text by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/1004
* feat(#1001): implement contact us and explore AI Agents buttons by @Jshen123 in https://github.com/reworkd/AgentGPT/pull/1006
* 👍 Sidebar updates by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1008
* 🐼 Updated the gradient by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/1005
* feat(#1007): implement backing component by @Jshen123 in https://github.com/reworkd/AgentGPT/pull/1010
* 🚀 Workflow Execution Engine by @awtkns in https://github.com/reworkd/AgentGPT/pull/992
* ⬆️ Bump tough-cookie from 4.1.2 to 4.1.3 in /next by @dependabot in https://github.com/reworkd/AgentGPT/pull/1009
* ⬆️ Bump chalk from 5.2.0 to 5.3.0 in /cli by @dependabot in https://github.com/reworkd/AgentGPT/pull/950
* ⬆️ Bump simple-git from 3.19.0 to 3.19.1 in /cli by @dependabot in https://github.com/reworkd/AgentGPT/pull/951
* ⬆️ Bump pydantic from 1.10.9 to 1.10.11 in /platform by @dependabot in https://github.com/reworkd/AgentGPT/pull/1012
* fix: apply anti-aliasing to text by @Jshen123 in https://github.com/reworkd/AgentGPT/pull/1021
* 👞 Footer Layout by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/1019
* ✨ Small workflow updates by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1018
* ✨ Backend for block insert by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1033
* ➡️ 1027 navbar top right buttons by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/1034
* 🤫 Auth Redirects by @awtkns in https://github.com/reworkd/AgentGPT/pull/1036
* feat(#999): update navbar buttons by @Jshen123 in https://github.com/reworkd/AgentGPT/pull/1020
* ✨ Node block saving in the frontend by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1037
* ✨ Basic of getting workflows to save and execute by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1040
* ♻️ Move schemas to seperate package by @awtkns in https://github.com/reworkd/AgentGPT/pull/1041
* ✨ Basic Slack Webhook block by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1042
* 🔥 Front end inputs by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1043
* 🔥 Ability to update input values on the Front end by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1044
* ✨ 1023 Create a card Component  by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/1038
* 🌟 Primary Button Glow by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/1048
* 🔥 Display block outputs in the frontend by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1052
* 🔥 Make autocomplete buttons fill out string template by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1054
* 🔧 Dynamic Consumer Groups by @awtkns in https://github.com/reworkd/AgentGPT/pull/1055
* 📱 Imrpove Responsiveness by @awtkns in https://github.com/reworkd/AgentGPT/pull/1061
* ✨ Enable string templating in the backend by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1057
* ✨ Workflow page updates by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1064
* feat(#1047): update navbar icon by @Jshen123 in https://github.com/reworkd/AgentGPT/pull/1066
* ✨ Navbar hover-highlight by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1068
* 🗑️ Workflow soft deletes by @awtkns in https://github.com/reworkd/AgentGPT/pull/1069
* 💅 Styling updates by @asim-shrestha in https://github.com/reworkd/AgentGPT/pull/1071
* 🌟 1067 add multiple cards and sliders by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/1070
* fix: landing page card text by @Jshen123 in https://github.com/reworkd/AgentGPT/pull/1083
* added color gradient to 'AI' on landing by @shahrishabh7 in https://github.com/reworkd/AgentGPT/pull/1085
* Feat/add gradient to landing page hero title by @Jshen123 in https://github.com/reworkd/AgentGPT/pull/1087
* Blog test by @jasangill1 in https://github.com/reworkd/AgentGPT/pull/1086
* Added left chevron on landing page 🔙 by @shahrishabh7 in https://github.com/reworkd/AgentGPT/pull/1089

**Full Changelog**: https://github.com/reworkd/AgentGPT/compare/v.0.9.5-beta...v.1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/reworkd/AgentGPT/releases/tag/v.0.10.0-beta'>v.0.10.0-beta</a>.</em>

**Comment by yhyu13** [comment]
Schedule for next release? It has been a month since previous beta release

**Comment by caonimashabimanong** [comment]
shit agentgpt project，fuck u to death ，fuck u fuck yo mom ，decay bitch！！！！！！！！！fuck u bitch self called IT genius bitch assd ugly ass IT bitch
